src/graphics/render_window.rs

```diff
@@ -289,6 +530,18 @@ impl RenderWindow {
     /// # Arguments
     /// * enabled - true to enable, false to disable
     ///
+    /// # Usage Example
+    ///
+    /// ```no_run
+    /// # use sfml::window::{Event, Style};
+    /// # use sfml::graphics::RenderWindow;
+    /// # // Create a new window with SFML window as name
+    /// # let mut window = RenderWindow::new((800, 600),
+    /// #                              "SFML window",
+    /// #                              Style::CLOSE,
+    /// #                              &Default::default());
+    /// window.set_key_repeat_enabled(true);
```

Comment:
I think my problem with a lot of these examples is that they are so simple, they are completely self-explanatory, and redundant.
It's already very apparent from the function signature that you can call it with either `false` or `true`, it really needs no dedicated example.
Adding tons of small examples here with a lot of setup and teardown (creating `RenderWindow`, etc.), just makes testing take a lot longer than it needs to be. Costing more time and energy.

I honestly believe small self-explanatory examples like this are unnecessary and can be removed.

Reply:
I was debating this in my head. I decided to look at the official rust docs and they used small redundant examples. I just copied what the official rust docs did.

I kind of agree with you though. I will get rid of the small self-explanatory examples then.